target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -456,6 +457,7 @@ ath79_setup_macs()
 		wan_mac=$(mtd_get_mac_binary factory 0x0)
 		lan_mac=$(macaddr_setbit_la "$wan_mac")
 		;;
+	engenius,emr3000|\
```

Comment:
Please move the block according to alphabetic sorting.

Reply:
Thanks for comment.